beater/test_approved_es_documents/TestPublishIntegrationErrors.approved.json

```diff
@@ -25,52 +25,6 @@
                     "my_key": 1,
                     "some_other_value": "foo bar"
                 },
-                "request": {
-                    "body": "Hello World",
-                    "cookies": {
-                        "c1": "v1",
-                        "c2": "v2"
-                    },
-                    "env": {
-                        "GATEWAY_INTERFACE": "CGI/1.1",
-                        "SERVER_SOFTWARE": "nginx"
-                    },
-                    "headers": {
-                        "array": [
-                            "foo",
-                            "bar",
-                            "baz"
-                        ],
-                        "content-type": "text/html",
-                        "cookie": "c1=v1; c2=v2",
-                        "some-other-header": "foo",
-                        "user-agent": "Mozilla Chrome Edge"
-                    },
-                    "http_version": "1.1",
-                    "method": "POST",
-                    "socket": {
-                        "encrypted": true,
-                        "remote_address": "12.53.12.1"
-                    },
-                    "url": {
-                        "full": "https://www.example.com/p/a/t/h?query=string#hash",
-                        "hash": "#hash",
-                        "hostname": "www.example.com",
-                        "pathname": "/p/a/t/h",
-                        "port": "8080",
-                        "protocol": "https:",
-                        "raw": "/p/a/t/h?query=string#hash",
-                        "search": "?query=string"
-                    }
-                },
-                "response": {
-                    "finished": true,
-                    "headers": {
-                        "content-type": "application/json"
-                    },
-                    "headers_sent": true,
-                    "status_code": 200
```

Comment:
Where did this `headers_sent` and `status_code` go?

Reply:
this was a typo in the mapping, fixed now. good catch :+1: